Source/core/CyclicBuffer.h

```diff
@@ -175,7 +175,7 @@ namespace Core {
         }
         inline bool IsValid() const
         {
-            return (_buffer.IsValid());
+            return (_buffer.IsValid() && (_administration != nullptr));
```

Comment:
Can we not make the buffer invalid if there no administration _administration == nullptr ? Otherwise I guess we need more checks throughout the code if the administratin can be null but we do have a valid buffer..
Or, if we can not invalidate the buffer if there is ni _administration, only base our Valid object on the _administration.
I think it is good to have 1 source of thruth and not 2...